RS/2D vector space/main.py

```diff
@@ -0,0 +1,137 @@
+import pygame, sys, time, random, math
+from pygame.locals import *
+
+pygame.init()
+pygame.mixer.init()
+
+# Declare all variables etc
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+
+windowWidth = 256
+windowHeight = 256
+
+tailLength = 20
```

Comment:
Change to the [PEP-8 style](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) for variable names: `lower_case_with_underscores`